blog/2024-06-01-barcamp-registration.md

```diff
@@ -0,0 +1,19 @@
+---
+author: Antonin Delpeuch
+title: "Register to participate to our BarCamp remotely"
+---
+
+Our first ever BarCamp is coming up in two weeks, on June 17th-19th, in Berlin and online.
+[Register now](https://barcamps.eu/openrefine-2024-barcamp/) to participate, whether you want to be onsite or remotely.
+This will give you access to the video conference room, let you propose and vote on sessions.
+
+The event will start Monday afternoon:
+* 14:00 CEST (12:00 UTC): Introductions round and opening of the event.
+* 14:30 CEST (12:30 UTC): Collaborative review and scheduling of [the proposed sessions](https://barcamps.eu/openrefine-2024-barcamp/sessions).
+* 16:00 CEST (14:00 UTC): Break
+* 16:30 CEST (14:30 UTC): Allocation of volunteering shifts (session facilitation, note-taking, Zoom caretaking)
+
+The sessions themselves will then take place on Tuesday and Wednesday, following the schedule established on Monday.
+One exception is the session about the mission, visions and values of OpenRefine facilitated by BoCoup, taking place on Tuesday at XX CEST (XX UTC), already scheduled because of the external facilitation.
```

Comment:
@magdmartin when is the BoCoup session again?

Reply:
The session will be on  June 18th. We still need to agree on the exact time; we are looking for a two hours between 14:00 CEST and 18:00 CEST. The exact start time will depend on what fit best the remote attendees.